components/test_pd_client/src/pd.rs

```diff
@@ -1185,7 +1188,31 @@ impl TestPdClient {
     }
 
     pub fn must_remove_peer(&self, region_id: u64, peer: metapb::Peer) {
-        self.remove_peer(region_id, peer.clone());
+        let mut region = None;
```

Comment:
Why not update the case instead? I'm afraid it accidentally breaks existing test cases.